appbench/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
+- sig-scalability-reviewers
+  
+approvers:
+- sig-scalability-approvers
+- jberkus
+- aakarshg
+- tduzan
```

Comment:
Apparently our presubmits are complaining OWNERs file - please fix.

Reply:
looks like @aakarshg and @tduzan need to become Kubernetes Org members.  I'll have to work on that.